tools/azure-sdk-tools/ci_tools/environment_exclusions.py

```diff
@@ -52,7 +52,7 @@ def is_check_enabled(package_path: str, check: str, default: Any = True) -> bool
     if package_path == ".":
         package_path = os.getcwd()
 
-    ci_enabled = get_config_setting(package_path, "ci_enabled", default)
+    ci_enabled = get_config_setting(package_path, "ci_enabled", True)
```

Comment:
This is a bug that luckily never reared it's ugly head before finding it. We absolutely want the default for `ci_enabled` to be `True` and should not rely on whatever's passed into this function. Luckily, everywhere we use this function, we either don't pass a `default` or do pass `True`.

Reply:
Good catch thanks Krista!